dockerfiles/hpcc/templates/_helpers.tpl

```diff
@@ -40,44 +42,14 @@ Create chart name and version as used by the chart label.
  {{- end -}}
 {{- end -}}
 
-{{/*
-Generate local config info into config section
-*/}}
-{{- /* Pass in a dictionary with component and me defined */ -}}
-{{- define "hpcc.generateComponentConfigMap" -}}
-{{- .me.name -}}.yaml: |
-  version: 1.0
-  {{ .component }}:
-{{ toYaml .me | indent 4 }}
-{{- end -}}
-
 {{- /* Generate a ConfigMap for a component */ -}}
 {{- /* Pass in a dictionary with root, component and me defined */ -}}
-{{- define "hpcc.generateConfigMap" -}}
-kind: ConfigMap 
-apiVersion: v1 
-metadata:
-  name: {{ .me.name }}-configmap 
-data:
+{{- define "hpcc.generateConfigMap" }}
```

Comment:
is this now really generateGlobalConfigMap?

Reply:
It's going to go as soon as we agree what to do about global options